cristin-import/build.gradle

```diff
@@ -54,6 +57,10 @@ dependencies {
 
 }
 
+test {
+    environment "MAX_SLEEP_TIME", "1"
```

Comment:
If writing to dynamo fails the handler will sleep for some time. But we do not want to sleep for a long time during testing.